src/router.php

```diff
@@ -2,25 +2,51 @@
 
 namespace Bugcache;
 
-use Aerys as aerys;
+use Aerys;
```

Comment:
Yes, should be consistent, hence I propose aliasing them. Makes it easier and look better.